tests/sentry/db/postgres/lookups/test_array_element_contains.py

```diff
@@ -0,0 +1,30 @@
+from unittest.mock import Mock, patch
+
+from sentry.db.postgres.lookups import ArrayElementContainsLookup
+
+
+def test_as_sql_basic_usage():
+    with (
+        patch(
+            "sentry.db.postgres.lookups.array_element_contains.ArrayElementContainsLookup.process_lhs"
+        ) as mock_process_lhs,
+        patch(
+            "sentry.db.postgres.lookups.array_element_contains.ArrayElementContainsLookup.process_rhs"
+        ) as mock_process_rhs,
+    ):
+        lhs = "column_name"
+        rhs = "%s"
+        mock_process_lhs.return_value = (lhs, [])
+        mock_process_rhs.return_value = (rhs, ["value"])
+
+        lookup = ArrayElementContainsLookup(lhs, rhs)
+        sql, params = lookup.as_sql(Mock(), Mock())
+
+        assert (
+            sql
+            == """EXISTS (
+            SELECT * FROM UNNEST(column_name) AS elem
+            WHERE elem LIKE '%%' || %s || '%%'
+        )"""
+        )
+        assert params == ["value"]
```

Comment:
this test doesn't actually test anything -- it's a copy paste of the implementation with mocks inserted.  the implementation could change in a compatible way and the test would break even if the functionality still worked

this should test the _behaviour_ and not the _implementation_

good rule of thumb: if you find yourself reaching for mock, mock, mock at the beginning and asserting exactly the implementation return value at the end: it's probably not a good test!